lib/logstash/codecs/json_lines.rb

```diff
@@ -53,7 +53,8 @@ def encode(event)
   # from_json_parse uses the Event#from_json method to deserialize and directly produce events
   def from_json_parse(json, &block)
     LogStash::Event.from_json(json).each { |event| yield event }
-  rescue LogStash::Json::ParserError
+  rescue LogStash::Json::ParserError => e
+    @logger.error("JSON parse error, original data now in message field", :error => e, :data => json)
```

Comment:
Since this condition is automatically recovered by Logstash, I don't think logging it as an error is the right choice. Maybe info? or warn?